enzyme/Enzyme/Runtime/EnzymeMPFR.h

```diff
@@ -0,0 +1,122 @@
+//===- EnzymeMPFR.h - MPFR wrappers ---------------------------------------===//
```

Comment:
Yeah it would be nice to have this in a file, but still get imported via tablegen, if you have a chance to set that up

Reply:
I could not make cmake properly handle the dependency between the included file and the .td so I inlined it the same way as the other headers.

Reply:
After hacking around with custom header importing I am thinking that the cleanest solution is for us to properly provide header file installation and ship them, what do you think?

Another option is to compile the mpfr wrappers in a static library and ship that.

Reply:
I'm okay either way. Pro of havng integrated header means that users of Enzyme as a library have definitional guarantees about semantics of the header/verion/custom syntax inside. On other hand, seeing code in include file can also be nice

Reply:
Maybe leave it as is for now, and we can adjust later?